pyls/plugins/jedi_completion.py

```diff
@@ -130,6 +132,12 @@ def _format_completion(d, include_params=True):
         'insertText': d.name
     }
 
+    if d.type == 'path':
+        path = osp.normpath(d.name)
+        path = path.replace('\\', '\\\\')
+        path = path.replace('/', '\\/')
```

Comment:
Do these replacements change the way you write paths on Windows and Posix systems in the Spyder editor?

Reply:
They just escape forward and backward slashes in order to prevent conflicts with the LSP snippets grammar. We're escaping them back in Spyder as well.

Reply:
Also, paths are normalized in order to preserve backward slashes in Windows and forward ones on POSIX